web/src/main/java/com/softserve/osbb/controller/VoteController.java

```diff
@@ -45,67 +51,86 @@
         User user = userService.findOne(vote.getUser().getUserId());
         user.getVotes().add(vote);
         vote.setUser(user);
-        for(Option option: vote.getOptions()) {
+
+        for (Option option : vote.getOptions()) {
             option.setVote(vote);
         }
-        vote = voteService.addVote(vote);
-        return new ResponseEntity<>(addResourceLinkToVote(vote), HttpStatus.OK);
+
+        return new ResponseEntity<>(
+                addResourceLinkToVote(voteService.addVote(vote)),
+                HttpStatus.OK);
     }
 
     @RequestMapping(value = "/{id}", method = RequestMethod.GET)
-    public ResponseEntity<Resource<Vote>> getVoteById(@PathVariable("id") Integer id) {
+    public ResponseEntity<Resource<Vote>> getVoteById(
+            @PathVariable("id") Integer id) {
         logger.info("Get vote by id: " + id);
-        Vote vote = voteService.getVoteById(id);
-        return new ResponseEntity<>(addResourceLinkToVote(vote), HttpStatus.OK);
+        return new ResponseEntity<>(
+                addResourceLinkToVote(voteService.getVoteById(id)),
+                HttpStatus.OK);
     }
 
-   @RequestMapping(value="/all/{osbbId}", method = RequestMethod.GET)
-   public ResponseEntity<List<Resource<VoteDTO>>> getAllVotesByOsbb(@PathVariable("osbbId") Integer osbbId) {
-       logger.info("Get all votes by osbbId: " + osbbId);
-       List<Vote> voteList = voteService.getAllVotesByDateOfCreation();
-       List<VoteDTO> voteDTOList = VoteDTOMapper.mapAllVoteEntityToDTO(
-               voteList.stream().filter(v -> v.getUser().getOsbb().getOsbbId() == osbbId)
-                       .collect(Collectors.toList()));
-       final List<Resource<VoteDTO>> resourceVoteList = new ArrayList<>();
-       for(VoteDTO v: voteDTOList) {
-           resourceVoteList.add(addResourceLinkToVote(v));
-       }
-       return new ResponseEntity<>(resourceVoteList, HttpStatus.OK);
-   }
-
-    @RequestMapping(value="/close/{osbbId}", method = RequestMethod.GET)
-    public ResponseEntity<Resource<Vote>> closeVote(@PathVariable("osbbId") Integer osbbId) {
+    @RequestMapping(value = "/all/{osbbId}", method = RequestMethod.GET)
+    public ResponseEntity<List<Resource<VoteDTO>>> getAllVotesByOsbb(
+            @PathVariable("osbbId") Integer osbbId) {
+        logger.info("Get all votes by osbbId: " + osbbId);
+        List<Vote> voteList = voteService.getAllVotesByDateOfCreation();
```

Comment:
try to avoid initializing list in this way.
as explained before - you can get null here